stock_return_request/models/stock_move.py

```diff
@@ -24,6 +24,7 @@ class StockMove(models.Model):
     def _compute_qty_returnable(self):
         """Looks for chained returned moves to compute how much quantity
            from the original can be returned """
+        self.qty_returnable = 0
```

Comment:
```suggestion
        moves_to_update = self.filtered(lambda x: x.state not in ["draft", "cancel"])
        (self - moves_to_update).qty_returnable = 0
        for move in moves_to_update:
        (...)
```

Reply:
This avoid to trigger update on the whole recordset if not necessary

Reply:
AFAIK, since v13, this is not a problem, as this doesn't retrigger a recomputation.